protocol/messenger_test.go

```diff
@@ -874,6 +874,67 @@ func (s *MessengerSuite) TestRetrieveTheirPrivateGroupChat() {
 	s.Require().NotNil(actualChat.LastMessage)
 }
 
+// Test receiving a message on an existing private group chat
+func (s *MessengerSuite) TestChangeNameGroupChat() {
+	var response *MessengerResponse
+	theirMessenger := s.newMessenger(s.shh)
+	response, err := s.m.CreateGroupChatWithMembers(context.Background(), "id", []string{})
```

Comment:
Maybe rename `"id"` to `"old-name"` to make it clear that second argument is the group chat name.